tower1:
  x: 0.4
  y: 0.4
  z: 0.81
  qx: 0.0027679497733095917
  qy: -0.7071029533482281
  qz: 0.002866580215573647
  qw: 0.70709938094877
